### DISCUSSION TITLE
Is dzsave still supported in version 8.14.2?

### DISCUSSION THREAD

**Discussion by sawpool** [libvips/libvips#3452] [Q&A]
Thank you for the vips command line tool which I used on Win10 [ 8.12.2] to create an image pyramid for DeepZoom. 

However upgrading to 8.14.2 the same command produces the response  'vips: unknown action "dzsave"'. 

I note that the command vips -l no longer lists dzsave in its output but there is nothing in the changelog to suggest dzsave has been removed.   I was able to revert to 8.12.2 to produce the required  image pyramid. 

Should I be using an action other than dzsave to create  image pyramids with 8.14.2?  Thanks in advance
Ray

**Comment by lovell** [comment]
Please see https://github.com/libvips/libvips/issues/3354
